src/towncrier/newsfragments/538.feature.rst

```diff
@@ -0,0 +1 @@
+``build`` now treats a missing fragments directory the same as an empty one, consistent with other operations.
```

Comment:
Is this a feature or a bug ?  Just asking.

Based on the test docstring .

Reply:
My rationale, even if it was a subtly indicated behavior, is that it's for most users a change in the expected behavior. I also could see it considered a bugfix.

It could also theoretically be considered a breaking change for users reliant on the current behavior (seems unlikely). I'll go with whatever scope the maintainers wish.

Since the latest recommendation is for bugfix, I'll switch to that, but with openness to change.